.travis.yml

```diff
@@ -15,7 +15,7 @@ before_cache:
   - ./ci/clear_cache.sh
 
 rust:
-  - 1.40.0 # min supported
+  - 1.43.0 # min supported
```

Comment:
I believe the compiler bug was fixed in `1.42.0`

Reply:
Could you please update the readme's stated msrv as well? There's a shield stating "1.40+"